scripts/TC4 - Aspects.zs

```diff
@@ -1307,9 +1512,55 @@ mods.thaumcraft.Aspects.set(<MagicBees:miscResources:5>, "praecantatio 0, bestia
 mods.thaumcraft.Aspects.set(<MagicBees:miscResources:6>, "praecantatio 1, bestia 1, alienis 1");
 mods.thaumcraft.Aspects.set(<MagicBees:miscResources:3>, "mortuus 0, exanimis 0, venenum 0, infernus 0");
 mods.thaumcraft.Aspects.set(<MagicBees:miscResources:4>, "mortuus 1, exanimis 1, venenum 1, infernus 1");
-mods.thaumcraft.Aspects.set(<MagicBees:capsule.magic:*>, "praecantatio 1");
-mods.thaumcraft.Aspects.set(<MagicBees:capsule.void:*>, "metallum 1");
```

Comment:
Why is this reverting to its non-wildcard format?

Reply:
> Unfortunately, some items cannot be assigned like this or they will cause the script to not load and throw an error that I can assume is some sort of outbounds? (a seemingly random integer not associated to error codes)

I'll have to test this more in-depth and maybe make a commit for that

Reply:
> > Unfortunately, some items cannot be assigned like this or they will cause the script to not load and throw an error that I can assume is some sort of outbounds? (a seemingly random integer not associated to error codes)
> 
> I'll have to test this more in-depth and maybe make a commit for that

From my testing Magic bees capsules, TiCon patterns, jerky, berrys, and mekanism plastic all triggered this. I don't really know a reason unless it something more fundamental to the mod.